src/fabric/include/fabric2.hrl

```diff
@@ -81,3 +82,7 @@
 
 
 -define(DEFAULT_BINARY_CHUNK_SIZE, 100000).
+
+%
+-define(DB_EXPIRATION_JOB_TYPE, <<"db_expiration">>).
```

Comment:
If these are just for testing, might be better to keep in them in the module. In the test we can use the binary values directly. If we ever change them the tests should fail right away and we'd know